server/controllers/items/lib/items.coffee

```diff
@@ -60,6 +60,17 @@ module.exports = items_ =
     .then FilterWithImage(assertImage)
     .map filterPrivateAttributes(reqUserId)
 
+  publicByLangAndDate: (limit, offset, lang, assertImage, reqUserId)->
```

Comment:
duplictation with `publicByDate` which is only used twice in the system, dont you think we can delete it and integrate it within `publicByLangAndDate`

Reply:
we could, but in a second movement in my opinion: I tried to think of a view that could be used by both but could find any, so to make `last-public` use the `publicByLangAndDate` view, we need to make it filter by lang too. Which in turn would imply updating the client `/inventory/last` layout. But once we are done with our use case of `recent-public` on the `/welcome` layout, we could have a look at how to merge those two endpoints.

In general, I prefer to start by duplicating, then progressively push everything in direction of the new implementation

Reply:
"Which in turn would imply updating the client " one more pro for the [git repo merge](https://github.com/inventaire/inventaire/issues/120)
indeed its not that easy (yet) lets see later